src/utils/query.js

```diff
@@ -26,7 +26,7 @@ export const handleResponse = (endpoint, res, placeholder) => {
 			}
 			throw new Error(`No ${endpoint} found using specified parameters.`);
 		}
-		return res.data[0];
+		return res.data;
```

Comment:
wouldn't this affect other queries?

Reply:
It was not returning a list of transactions from API so I had to change but I also tested all the commands which are using it and it works fine and also all the tests are passing.

Reply:
I think test passes because we stub the API response, but it changes the format ='(
```
> ./bin/run account:get 123L,456L                                                             
[
        {
                "0": {
                        "address": "123L",
                        "unconfirmedBalance": "6122216612",
                        "balance": "6122216612",
                        "publicKey": "",
                        "secondPublicKey": ""
                }
        },
        {
                "0": {
                        "address": "456L",
                        "unconfirmedBalance": "1000",
                        "balance": "1000",
                        "publicKey": "",
                        "secondPublicKey": ""
                }
        }
]
```